img/e/epidemicsound.com.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 65 80" xmlns="http://www.w3.org/2000/svg"><path d="M13.478 13.739C-4.466 32.03-4.492 61.673 13.397 80h27.891L27.424 65.866c-10.269-10.469-10.269-27.442 0-37.91a25.946 25.946 0 0 1 37.191 0V3.986C47.74-3.605 27.305-.356 13.478 13.74Zm51.137 14.216V60.72H32.473l32.142-32.765Z" fill="#FFF" fill-rule="evenodd"/></svg>
```

Comment:
I think this logo works better as the white logo without a background potentially could have a contrast accessibility issue.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" style="background:#000;fill:#fff" viewBox="0 0 100 100"><path d="M38 33a30 30 0 0 0 0 42h18l-9-9c-7-6-7-17 0-24a16 16 0 0 1 23 0V27c-10-4-23-2-32 6zm32 9v21H50Z"/></svg>
```
![](https://camo.githubusercontent.com/e2481ea710d24a24a71feb635725fec7d9386aa346df7efaa87cf5a458e79d05/68747470733a2f2f642e74656d7066696c65732e646f776e6c6f61642f44363234323238414144463042372f3f703d646d50726f7843624d6a70685944577434)

Reply:
Their [press kit](https://downloads.ctfassets.net/ojtnytzl1djm/1581vv70ZjI3Wm9drSbxJP/c6169971636f9585211fd7e554297cfb/2021-epidemic-sound-press-kit.zip) uses a simple black logo:
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 378.7 459.5"><path d="M271.7 0A269.3 269.3 0 0 0 78.5 459.5H242l-81.3-81.2a153.9 153.9 0 0 1 0-217.7 154.2 154.2 0 0 1 218 0V22.9c-34-15-70.5-22.6-107-22.9Zm107 160.6L190.3 348.8h188.4z"/></svg>`